### ISSUE TITLE
Upload PRs to Nimbledroid for Automated Performance Testing

### ISSUE THREAD

**Issue by colintheshots** [mozilla-mobile/fenix#4512]
We want to get actionable information about the performance impact of major, new features and code refactors. If we had the ability to produce Nimbledroid reports from PRs, we would have the information we need.

We'll need to be careful not to exceed our monthly upload limit at Nimbledroid. We may want to only run these tests on demand or only on the initial pull request, but not every update. 

Update: Based on initial feedback from the performance team, we may need to trigger Nimbledroid runs based upon adding a label to a PR. This may increase the estimate, since a Github webhook will need to be written to do so.

**Comment by liuche** [comment]
Will wait on bors-ng.

**Comment by hawkinsw** [comment]
I am working with @JohanLorenzo on getting TC to run on the machine where I take nightly performance regression numbers (#5020 ). If/when we get TC running, we should be able to monitor for pushes to the code and run on-demand profiles on actual hardware and have the artifacts available publicly. Would that potentially "solve" this problem?

**Comment by hawkinsw** [comment]
https://developer.github.com/v3/guides/building-a-ci-server/

**Comment by hawkinsw** [comment]
https://nimbledroid.com/help/ci

**Comment by mcomella** [comment]
(copy-paste) Not a priority: closing.
